pandas-stubs/core/series.pyi

```diff
@@ -1580,6 +1585,51 @@ class Series(IndexOpsMixin[S1], NDFrame):
     # just failed to generate these so I couldn't match
     # them up.
     @overload
+    def __add__(
+        self: Series[Never],
+        other: num | _str | timedelta | Timedelta | _ListLike | Series | np.timedelta64,
+    ) -> Series: ...
+    @overload
+    def __add__(self: Series[int], other: Series[Never]) -> Series: ...
```

Comment:
Is it possible to replace this overload with `def __add__(self: Series, other: Series[Never]) -> Series: ...` (removing the `int` part)? 

Then we can also remove the other Never overlords for float/complex